src/chessground.ts

```diff
@@ -7,10 +7,10 @@ import * as events from './events';
 import { render, renderResized, updateBounds } from './render';
 import * as svg from './svg';
 import * as util from './util';
+import { renderPockets, renderPocketsInitial } from './pocket';
 
-export function Chessground(element: HTMLElement, config?: Config): Api {
+export function Chessground(element: HTMLElement, pocketTop?: HTMLElement, pocketBottom?: HTMLElement, config?: Config): Api {
```

Comment:
The pockets should be after the config so that there's no need to change the old code in some places. Also, if a site that doesn't have any pocket variants wants to use chessgroundx in the future, they won't need to care about these arguments.
```suggestion
export function Chessground(element: HTMLElement, config?: Config, pocketTop?: HTMLElement, pocketBottom?: HTMLElement): Api {
```